client/test/io.js

```diff
@@ -0,0 +1,11 @@
+/**
+  * Mock class for io
+  */
+
+module.exports = () => ({
+  socket: {
+    postAsync: function () {
+      return Promise.resolve().then(() => ({}));
```

Comment:
Is it worth adding a case for failure/requests that take awhile? For example, we could verify that the toast doesn't go away until a response from the server is received, but it does go away if an error occurs.